zuul.d/jobs.yaml

```diff
@@ -894,6 +895,7 @@
     vars:
       ansible_test_python: 3.8
       ansible_test_command: units
+      ansible_coverage_report: "ansible-test coverage report"
```

Comment:
lets move this into playbooks, I'd prefer not to have cmd syntax in jobs directly